### PR TITLE
Show research disclaimer only on Lagunita register

### PR DESCRIPTION
**Pull request by stvstnfrd**
Instead of telling the template _that_ theming is enabled,
let's actually tell it _which_ one is.

### CODE REVIEW

**Comment by stvstnfrd** [comment]
Please review: @caesar2164

CC: @caseylitton 

This came up during the CME work. I'm teasing it out into a separate PR since it affects code already on master.

File: lms/templates/student_account/register.underscore [hunk 1]
```diff
@@ -58,7 +58,7 @@
 
     <%= context.fields %>
 
-    <% if (context.isNonMicrositeTheme) { %>
+    <% if (context.themeName === 'lagunita') { %>
```
Comment (caesar2164) [inline lms/templates/student_account/register.underscore]: are you positive this is the only place that relies on this anymore?

Comment (caesar2164) [inline lms/templates/student_account/register.underscore]: as in is there anything else that looks for `isNonMicrositeTheme` in platform?

Comment (stvstnfrd) [inline lms/templates/student_account/register.underscore]: Yes, we also grepped this f2f.

**Comment by caesar2164** [comment]
from F2F, all instances of `isNonMicrositeTheme` are now gone!

File: lms/templates/student_account/register.underscore (hunk 1 above)
Comment (caesar2164) [inline lms/templates/student_account/register.underscore]: awesome

**Comment by caesar2164** [comment]
👍

File: openedx/stanford/lms/envs/common.py [hunk 2]
```diff
@@ -108,6 +108,7 @@
     # Include TEMPLATE_VISIBLE_SETTINGS in templates
     'settings_context_processor.context_processors.settings',
 ]
+THEME_NAME = 'lagunita'
```
Comment (stvstnfrd) [inline openedx/stanford/lms/envs/common.py]: @caesar2164 Tests fail without some value set.

**Pull request merged**